resources/user-agents/browsers/whale/whale.json

```diff
@@ -79,6 +79,33 @@
           ]
         }
       ]
-    }
+    },
+    {
+        "userAgent": "Whale Browser for Android",
+        "browser": "whale browser",
+        "platform": "Android",
+        "engine": "Blink",
+        "device": "general Mobile Device (Touch)",
+        "properties": {
+          "Parent": "DefaultProperties",
+          "Comment": "Whale Browser #MAJORVER#.#MINORVER#",
+          "Version": "#MAJORVER#.#MINORVER#"
+        },
+        "children": [
+          {
+            "match": "Mozilla/5.0 (#PLATFORM#) AppleWebKit/* (*khtml*like*gecko*) Chrome/* Whale/#MAJORVER#.#MINORVER#* Mobile Safari/*",
+            "device": "general Mobile Phone (Touch)",
+            "platforms": [
+              "Android"
+            ]
+          },
+          {
+            "match": "Mozilla/5.0 (#PLATFORM##DEVICE#) AppleWebKit/* (*khtml*like*gecko*) Chrome/* Whale/#MAJORVER#.#MINORVER#* Mobile Safari/*",
```

Comment:
Please remove this rule.

The `#DEVICE#` token without defined devices leads to errors. Without this token its the same as the rule above.

Reply:
Thank you for your review. I have corrected that you pointed.